config/local/puppet/modules/profile/templates/openondemand/common/ondemand.d/support_ticket_snow.yml.erb

```diff
@@ -0,0 +1,17 @@
+---
+support_ticket:
+  servicenow_api:
+    server: "<%= @server %>"
+    map:
+      u_email_source: "email"
+      watch_list: "cc"
+    payload:
+      # "guest" = 5136503cc611227c0183e96598c4f706
+      caller_id: "5136503cc611227c0183e96598c4f706"
+      # "troubleshooting" = 4e7017196fc5c100a54fa981be3ee4c9
+      u_category: "4e7017196fc5c100a54fa981be3ee4c9"
+      contact_type: "External System"
+      #"FAS RC - IQSS Support" = 091302291b9302509ebaca262a4bcb21
+      assignment_group: "091302291b9302509ebaca262a4bcb21"
+      #"FAS Research Computing Services > IQSS Support > IQSS Triage" = b63f3ded1b5302509ebaca262a4bcbfe
+      u_business_service: "b63f3ded1b5302509ebaca262a4bcbfe"
```

Comment:
shouldn't these hardcoded values be read from an env variables with the configuration instead?